pkg/kubelet/container/helpers.go

```diff
@@ -52,6 +52,7 @@ type RuntimeHelper interface {
 	GetPodCgroupParent(pod *v1.Pod) string
 	GetPodDir(podUID types.UID) string
 	GeneratePodHostNameAndDomain(pod *v1.Pod) (hostname string, hostDomain string, err error)
+	GetHostNameForKernel(hostname string, hostDomainName string, hostnameFQDN bool) (string, error)
```

Comment:
In other places, Hostname is used.
Why is HostName used here ?

Reply:
agree, i will change